frontend/src/screens/BackupNode.tsx

```diff
@@ -80,57 +77,57 @@ export function BackupNode() {
   return (
     <>
       <SettingsHeader
-        title="Migrate Your Node"
-        description="Your Alby Hub will be stopped and you will receive a backup file you can import on another host or machine."
+        title="Migrate Alby Hub"
```

Comment:
can we rename the component and file to `MigrateNode` rather than `BackupNode`? this was my mistake calling it that.

Reply:
refactored node-backup and node-backup-success to node-migrate

Reply:
also updated routes